src/__internal__/checkable-input/checkable-input.style.js

```diff
@@ -22,19 +22,13 @@ const StyledCheckableInputWrapper = styled.div`
     inputWidth,
     labelWidth,
     labelInline,
-    ml,
     reverse,
     theme,
   }) => css`
     ${FieldLineStyle} {
       display: flex;
     }
 
-    ${ml &&
```

Comment:
This component isn't using styled-system - are you sure we can remove this? Must have been a reason it was added

Reply:
Yeah it's a leftover from the times when we didn't use styled-system globally.